OWML.ModHelper.Menus/LayoutManager.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UI;
+using OWML.Common.Menus;
+using System.Collections.Generic;
+using OWML.ModHelper.Events;
+
+namespace OWML.ModHelper.Menus
+{
+    public class LayoutManager : ILayoutManager
+    {
+        private const int FontSize = 36;
+        private static readonly Vector2 NormalPivot = new Vector2(0.5f, 0.5f);
+
+        public LayoutGroup LayoutGroup { get; private set; }
+        public int ChildCount => LayoutGroup.transform.childCount;
+
+        private readonly UIStyleManager _styleManager;
+        private readonly ModUIStyleApplier _styleApplier;
+        private readonly Vector3 _scale;
+        private readonly HashSet<Graphic> _constantGraphics = new HashSet<Graphic>();
+        private readonly HashSet<Graphic> _backingGraphics = new HashSet<Graphic>();
+
+        public LayoutManager(LayoutGroup layout, UIStyleManager styleManager, ModUIStyleApplier styleApplier,
```

Comment:
This constructor isn't called, can it be removed?

Reply:
why do you want to remove all that potentionally useful functionality

Reply:
1. Not used -> not tested
2. More to maintain in the future
3. It's a slippery slope... 
@Raicuparta wanna break a tie?